test/Conversion/amd/tritongpu_wmma_dot_to_llvm.mlir

```diff
@@ -168,13 +169,14 @@ module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 4 : i32, "ttg.thr
 
 #shared = #ttg.shared<{vec = 1, perPhase = 1, maxPhase = 1, order = [2, 1, 0], hasLeadingOffset = false}>
 #mma1 = #ttg.amd_wmma<{version = 1, warpsPerCTA = [2, 1, 4]}>
+!memdesc0 = !ttg.memdesc<4x16x32xf16, #shared, #ttg.shared_memory>
 module attributes {"ttg.num-ctas" = 1 : i32, "ttg.num-warps" = 8 : i32, "ttg.threads-per-warp" = 32 : i32} {
   // CHECK-LABEL: wmma_dot_operand3d
-  tt.func @wmma_dot_operand3d(%arg0: !ttg.memdesc<4x16x32xf16, #shared>) {
+  tt.func @wmma_dot_operand3d(%arg0: !memdesc0) {
     // CHECK-COUNT-4: llvm.load %{{.*}} : !llvm.ptr<3> -> vector<16xf16>
-    %0 = ttg.local_load %arg0 : !ttg.memdesc<4x16x32xf16, #shared> -> tensor<4x16x32xf16, #ttg.dot_op<{opIdx = 0, parent = #mma1, kWidth = 16}>>
+    %0 = ttg.local_load %arg0 : !memdesc0 -> tensor<4x16x32xf16, #ttg.dot_op<{opIdx = 0, parent = #mma1, kWidth = 16}>>
```

Comment:
sorry for the back and forth on this but I wonder if we want to qlias the whole memdesc type, instead we could create an alias for the memory space only, that would make it more consistent with out tensor type representation. WDYT?

Reply:
Do you mean `#ttg.shared_memory` -> `shared_memory`?

Reply:
yeah `#ttg.shared_memory` -> `#smem`?

Reply:
OK

Reply:
Thought about it a bit more. 

I have two additional concerns:

1. Should we use attribute or type to represent the shared memory space? 
2. If we want to shorten the text of `shared_memory`, maybe we can just change its `mnemonic` to `smem`.

Reply:
> Should we use attribute or type to represent the shared memory space?

How could it be a type?

> If we want to shorten the text of shared_memory, maybe we can just change its mnemonic to smem.

Fair point, we would still have ttg.smem then? This fine too, I don't mean to bike shed so I'll leave it up to you